src/comments.js

```diff
@@ -161,6 +165,7 @@ function attach(comments, ast, text) {
           comment
         ) ||
         handleTryStatementComments(enclosingNode, followingNode, comment) ||
+        handleDecoratorComments(enclosingNode, comment) ||
```

Comment:
```ts
@AutoSubscribeStore
//store in resub, is a class extends from StoreBase.
class HelloStore extends StoreBase {
    @autoSubscribe
    // this method should return a string type
    getHello():string {
        return this._helloString;
    }
}
```
:arrow_down:
```ts
//store in resub, is a class extends from StoreBase.
@AutoSubscribeStore
class HelloStore extends StoreBase {
  @autoSubscribe // this method should return a string type
  getHello(): string {
    return this._helloString;
  }
}
```

But I see that it does not really handle the decorated methods well when I don't add a newline after the decorators.

I'll keep working on this, but I can only invest a couple of minutes per day since I'm actually on a vacation :smile: